### ISSUE TITLE
Exporter error: CustomActions Completion Report

### ISSUE THREAD

**Issue by gmaloneccg** [muccg/rdrf#1303]
When exporting ICHOMCRC from staging. Staging shows the Completion Report menu option for clinical staff and admin, present on the patient page. After export, Aloe testing demonstrates that the menu option for Completion Report does not exist on the patient data pages, for both clinical staff and admin users.

Some output from Aloe tests;
  - Admin;
```
AssertionError: Item "Completion Report" not found in menu "Menu".
Entries:
['Consents (ICHOM Colorectal Cancer)', 'Genes', 'Laboratories', 'Patient List', 'Reports', 'Users']
```
  - Clinical staff;
```
Exception: Menu item list:
['Patient List']
```

Output was provided with functions similar to the following (not exact, but used for example);
```python
# Case 1;
for item in find_multiple(xp):
    link_list.append(item.get_attribute("text").strip())
assert "Completion Report" in link_list,\
    "Item not found. List:\n%s" % link_list

# Case 2;
try:
    find(
        (
            "//a[contains(.,'%s')]/..//a[contains(.,'%s')]"
            % (menu, item)
        )
    )
except Nse:
    ...
```

**Comment by id2359** [comment]
Try opening the zip file in the aloe branch

it will show what's been loaded - the behaviour you're seeing is most likely due to the zip file containing an earlier version of CIC which doesn't have the completion report. If so , you'll need to recreate the test data ( let us know if you need help with that)

if the completion report IS in the zip file ( you'll need to grep through the json files for "Completion" to see ) something else is going wrong - let me know if this is the case and I'll check

**Comment by gmaloneccg** [comment]
Opened issue #1304 after a bit more testing.

**Comment by jithendralal** [comment]
The fix in #1304 now adds rdrf_customaction.json to the zip created locally, which has the "Completion Report" in it.
